### DISCUSSION TITLE
Tab title flickering prevention.

### DISCUSSION THREAD

**Discussion by LJISAMoige** [Woundorf/foxreplace#330] [Q&A]
I've been trying to stop tab titles from blinking ten times a second because that's insufferable.
I've set up to replace `([_a-z][a-z0-9_\s]*\.\s*)*document\s*.\s*title\s*=` by nothing in scripts, but no luck.
It is a regex, I've checked apply auto both on page load and every second, and replace scripts, but it doesn't seem to work?
If I f12 the pages and search their js code for that same regex I'll get the incriminated matches, here and untouched.
Is this not possible with this extension, or am I doing something wrong?

**Comment by Woundorf** [comment]
Hi, you have to go to the options page and enable the "Replace scripts" option.

**Reply by LJISAMoige** [reply]
I've checked apply auto both on page load and every second, and replace scripts.

**Reply by Woundorf** [reply]
If the script is in the head you have to set the HTML mode to input & output. Note that this can break some pages, but you can can try if it works for you and restrict it for this page..

**Reply by LJISAMoige** [reply]
The site I'm testing this on has the script referenced in the body of the html. I'm willing to try your solution, but I don't know how to do it. I'm using firefox as a browser. Could this issue be related? #309

**Reply by Woundorf** [reply]
I have created a basic example where the title is changed every second. Even if the script node is deleted the title keeps changing because there is a timer create with `window.setInterval` that remains active. Your only option is to redefine that function before it's called. With FoxReplace you can't because substitutions are applied on body load, but I think that with Greasemonkey you can a script a little [sooner](https://wiki.greasespot.net/Metadata_Block#.40run-at). With this you could just do `window.setInterval = function() {}`.
